.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: dart
 
 dart:
-  - 2.3.0
```

Comment:
It's good to keep this in sync w/ the lowerbound of the pubspec.yaml sdk constraint

Reply:
Good to know. Done.